backend/image/models.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+
+
+class Image(models.Model):
+    file = models.ImageField(upload_to='images/')
```

Comment:
Perhaps it's better to add a subfolder here for better overview, for example:  `uploads/%Y/%m/%d/`
This will also allow multiple files with the same name, while keeping the original filename in place.

Reply:
Also a good point. I've updated the upload path.